flink-rpc/flink-rpc-akka/src/test/java/org/apache/flink/runtime/rpc/akka/AkkaRpcServiceTest.java

```diff
@@ -371,7 +352,29 @@ private Collection<CompletableFuture<Void>> startStopNCountingAsynchronousOnStop
 
         countDownLatch.await();
```

Comment:
I cannot mark the two `assert` calls above, but I'm wondering whether we should move them underneath the `countDownLatch.await` call. :thinking: 

Don't we want to test that the RpcService is not terminated, yet, even after stop is called on all Endpoints? That is the case when the `countDownLatch` is triggered and its `await` leaves the blocking state.

Reply:
good point